tests/bsim/bluetooth/host/adv/chain/testcase.yaml

```diff
@@ -0,0 +1,11 @@
+tests:
+  bluetooth.host.adv.chain:
+    build_only: true
```

Comment:
> build_only: true

Is it planned that these will also eventually run the tests?

Reply:
:)
step 1) (what we already have) was to be able build and place the executable in the "right" place.
step 2) is to have dependencies between tests and images (so twister knows what images a test needs)
step 3) would be to also be able to run them from twister.

Reply:
bsim harness in twister as of now only supports building, and it also copies files to appropriate location. To run them, old shell scripts will still be used.